src/org_parser/transform.cljc

```diff
@@ -45,17 +46,31 @@
                    first
                    count)
         title (->> properties
-                   (property :title)
-                   (str/join " "))]
+                   (property :text))
+        tags (->>  []) ;; TODO extract tags from the last element of :text if it's a :text-normal; empty otherwise;
+                            ;; use regex #'\s+(:[a-zA-Z0-9_@#%]+)+:\s*$' (see EBNF), if it is matching remove it from the :text-normal, trim it and split by ':'
+        ]
     ;; a headline introduces a new headline
     (update state :headlines conjv {:headline {:level level
-                                               :title title}})))
+                                               :title title
+                                               ;;:tags  tags
+                                               ;; TODO much more to come, e.g. planning info (already parsed)
+                                               }})))
 
 
 ;; content-line needs to simply drop the keyword
 (defmethod reducer :content-line [state [_ ast] raw]
   (reducer state ast raw))
 
+;; Merge consecutive :text-normal inside a :text list. They come from
```

Comment:
@munen How to do this transformation:
`[:text [:text-normal "uiea"] [:text-normal "/foo"]]` => `[:text [:text-normal "uiea/foo"]]`
I laid out an fold algorithm but cannot implement it ATM.